packages/flutter_tools/lib/src/plugins.dart

```diff
@@ -789,11 +789,11 @@ import 'package:{{name}}/{{file}}';
 import 'package:flutter_web_plugins/flutter_web_plugins.dart';
 
 // ignore: public_member_api_docs
-void registerPlugins(PluginRegistry registry) {
+void registerPlugins(Registrar registrar) {
 {{#plugins}}
-  {{class}}.registerWith(registry.registrarFor({{class}}));
+  {{class}}.registerWith(registrar);
 {{/plugins}}
-  registry.registerMessageHandler();
+  ui.webOnlySetPluginHandler(registrar.handleFrameworkMessages);
```

Comment:
`handleFrameworkMessages` -> `handleFrameworkMessage`

Reply:
turns out i also need to change the import of "dart:ui" to "as ui".
(Any idea why we had the import before if it was unused?)